### DISCUSSION TITLE
gcp account

### DISCUSSION THREAD

**Discussion by hammadahmad23** [Guvi-MentorDE/BE_batch_Guvi_Aug6#49] [General]
Sir,

I have created new email Id and registered again on GCP as you have told me in the session today but I am facing same issue again in new account.

**Comment by Guvi-MentorDE** [comment]
hi @hammadahmad23  ,  Can you please follow the same steps provided in the link : https://www.youtube.com/watch?v=FPd-jC4QIPk. Let me know still the issue exists.
